### ISSUE TITLE
App can't be launched: illegal hardware instruction

### ISSUE THREAD

**Issue by sashahilton00** [bitgapp/new-eqmac-archive#14]
Have just tried installing eqMac, but not having any luck getting it running, it crashes on start. The error log is as follows:

```
/Applications/eqMac.app/Contents/MacOS/eqMac 
2020-01-25 16:13:20.485 eqMac[68725:3269946] Sentry Started -- Version: 4.4.0
Error: 561211770 ->  /Users/romanskisils/Programming/Bitgapp/eqmac/native/app/Source/Audio/Sources/System/Driver.swift:65

Error: 561211770 ->  /Users/romanskisils/Programming/Bitgapp/eqmac/native/app/Source/Audio/Sources/System/Driver.swift:99

ERROR: SentryCrashMachineContext.c (232): void sentrycrashmc_resumeEnvironment(): thread_resume (ffffffff): (ipc/send) invalid destination port
[1]    68725 illegal hardware instruction  /Applications/eqMac.app/Contents/MacOS/eqMac
```

This is running on 10.15.2, Macbook Pro (15") 2012. Let me know if there's any other info you want me to gather. Judging by the console output though, I imagine you probably have a more verbose log in the Sentry console.

**Comment by kerstinek** [comment]
That same issue on Mac mini (2018) with macOS 10.15.3 
```
2020-02-07 08:05:18.314 eqMac[1468:25709] Sentry Started -- Version: 4.4.0
Error: 561211770 ->  /Users/romanskisils/Programming/Bitgapp/eqmac/native/app/Source/Audio/Sources/System/Driver.swift:65

Error: 561211770 ->  /Users/romanskisils/Programming/Bitgapp/eqmac/native/app/Source/Audio/Sources/System/Driver.swift:99

ERROR: SentryCrashMachineContext.c (232): void sentrycrashmc_resumeEnvironment(): thread_resume (00009a07): (os/kern) invalid argument
Illegal instruction: 4
```

**Comment by eric-hu** [comment]
I ran into the same on my Macbook Pro 16" with macOS 10.15.3:

```
$ /Applications/eqMac.app/Contents/MacOS/eqMac ; exit;
2020-02-24 09:54:08.610 eqMac[1639:17261] Sentry Started -- Version: 4.4.0
Error: 561211770 ->  /Users/romanskisils/Programming/Bitgapp/eqmac/native/app/Source/Audio/Sources/System/Driver.swift:65

Error: 561211770 ->  /Users/romanskisils/Programming/Bitgapp/eqmac/native/app/Source/Audio/Sources/System/Driver.swift:99

ERROR: SentryCrashMachineContext.c (232): void sentrycrashmc_resumeEnvironment(): thread_resume (00009107): (ipc/send) invalid destination port
Illegal instruction: 4
logout
Saving session...completed.
Deleting expired sessions...4 completed.

[Process completed]
```

The app was working until I restarted my computer. Before the restart, it would require app restarts after waking from sleep.

**Comment by nodeful** [comment]
I have just released a new version (v0.0.1) of the app. Please download from the website or GitHub and test. This issue might be related to #16

**Comment by nodeful** [comment]
Hey everyone 👋 
I will be open sourcing the new version of eqMac, therefore, I am closing this issue and repository as I will be transfering [nodeful/eqMac2](https://github.com/nodeful/eqMac2) repo to [bitgapp/eqMac](https://github.com/bitgapp/eqMac)
Please bear with me for the next couple of days as I setup that new repo. The eqMac2 code will be living in the [old-eqMac2](https://github.com/bitgapp/eqMac/tree/old-eqmac2) branch if anyone is interested in referencing it for whatever reason.
Please try out the new version and report bugs to the new repo.
Thanks for using eqMac.
